### PR TITLE
feat: Add stacktrace on unexpected errors

### PR DESCRIPTION
**Pull request by taorepoara**
## Description of the changes
Add stacktrace in unexpected errors logs to have more context

## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
  - [ ] 👍 yes
  - [x] 🙅 no, because they aren't needed
  - [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes

- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [ ] 📓 docs.lenra.io
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by taorepoara** [comment]
@pichoemr It might not be the best way to do that. 

Maybe a method that returns a string of the error with the stacktrace appended would be better to use it in telemetry for example.

**Comment by taorepoara** [comment]
Also, I'm not sure that the stacktrace is added to the Sentry issues.

File: lib/channels/app_socket.ex [hunk 1]
```diff
@@ -78,7 +79,10 @@ defmodule ApplicationRunner.AppSocket do
             {:error, ErrorHelpers.translate_error(reason)}
 
           {:error, reason} ->
-            Logger.error(reason)
+            reason
+            |> Errors
```
Comment (Nesqwik) [inline lib/channels/app_socket.ex]: What is that ? Probably a fail :p
If not, It's the first time I see that in elixir.

**Review by Nesqwik — CHANGES_REQUESTED** [review]

File: lib/channels/app_socket.ex (hunk 1 above)
Comment (taorepoara) [reply lib/channels/app_socket.ex]: Totally !
My bad

**Pull request merged**